src/pymovements/gaze/experiment.py

```diff
@@ -107,12 +166,29 @@ def __init__(
             origin=origin,
         )
 
-        checks.check_is_not_none(sampling_rate=sampling_rate)
-        assert sampling_rate is not None
+        checks.check_is_mutual_exclusive(sampling_rate=sampling_rate, eyetracker=eyetracker)
+
+        self.eyetracker = eyetracker
+
+        self._sampling_rate = sampling_rate
 
-        checks.check_is_greater_than_zero(sampling_rate=sampling_rate)
+        checks.check_is_not_none(sampling_rate=self.sampling_rate)
+        assert self.sampling_rate is not None
 
-        self.sampling_rate = sampling_rate
+        checks.check_is_greater_than_zero(sampling_rate=self.sampling_rate)
+
+    @property
+    def sampling_rate(self) -> float | None:
+        """Get sampling rate of experiment."""
+        if self._sampling_rate is not None:
+            return self._sampling_rate
+
+        return self.eyetracker.sampling_rate
+
+    @sampling_rate.setter
+    def sampling_rate(self, _sampling_rate: float | None = None) -> None:
```

Comment:
Please rename the argument here to `sampling_rate` without the leading underscore.

We will only use the leading underscore for marking attributes as private.